assets/components/NotificationPopup.tsx

```diff
@@ -19,15 +19,11 @@ export interface IProps {
 }
 
 export const NotificationPopup = (props: IProps) => {
-    const today = (new Date()).toISOString().substring(0, 10);
-    const pausedFrom = props.fullUser.notification_schedule?.pause_from ?? '';
-    const pausedTo = props.fullUser.notification_schedule?.pause_to ?? '';
-    const notificationsArePaused = (
-        pausedFrom != '' && pausedFrom <= today &&
-        pausedTo != '' && pausedTo >= today
-    );
+    const today = getTodayTimestamp();
+    const pausedFrom = parseISODate(props.fullUser.notification_schedule?.pause_from);
+    const pausedTo = parseISODate(props.fullUser.notification_schedule?.pause_to);
 
-    if (notificationsArePaused) {
+    if (pausedFrom <= today && pausedTo >= today) {
```

Comment:
I'd be easier to understand if you used variables. Maybe `notificationsArePaused` or `notificationsWillBePaused` or whatever you're checking for in that if condition.